### PR TITLE
PR Review Best Practices

### PR DESCRIPTION
**Pull request by mwbenowitz**
This PR updates our page on PR review best practices. This page hasn't been touched in 5+ years so this includes both a general update as well as the addition of a few things:

- Added recommendation to use a `CODEOWNERS` file to help encourage a practice of tagging all team members in PRs
- Added general recommendation for timing and length of PR review processes
- Added recommendation to re-tag reviewers after updating a PR based on feedback to bump it back up in peoples' inboxes
- Generally reformatted

Please provide any feedback or additional best practices that you'd like to see added!

### CODE REVIEW

**Review by samanthaandrews — APPROVED** [review]

File: standards/peer-review.md
Comment (hokei) [inline standards/peer-review.md:3]: Why not MUST?

File: standards/peer-review.md
Comment (hokei) [inline standards/peer-review.md:14]: and/or request a code walkthrough/live review from the developer

File: standards/peer-review.md
Comment (mwbenowitz) [reply standards/peer-review.md:3]: Yeah, I'll pull that up and merge these two points. My thinking was that there are at times small teams where reviews are more difficult to come by and while our main effort should be to make that not happen, also this helps prevent blockage? Maybe unreviewed merging should be allowed only if there is a written and approved policy?

Comment (mwbenowitz) [reply standards/peer-review.md]: @hokei Updated this policy to include the requirement for a written policy. Also wondering if I should add a policy that if the requester specifically asks for permission to merge without a review and is granted? I know I have been asked for this because the engineer clearly could explain why the code was safe to merge without a review and I couldn't see any capacity to complete the review quickly.

File: standards/peer-review.md
Comment (nonword) [inline standards/peer-review.md:9]: It looks like we generally use user handles in our `CODEOWNERS`. Any experience/interest in tagging by team? Or are our write/admin teams usually larger than the set of handles we put in CODEOWNERS? Just seems a shame to have another thing to update for on/off-boarding, but given that git teams establish actual access, maybe that's fine.

**Review by nonword — COMMENTED** [review]
Much improved!

File: standards/peer-review.md
Comment (samanthaandrews) [reply standards/peer-review.md:9]: Usually the write/admin teams are larger than what we put in CODEOWNERS, but tbh I'm cool with you managing your subteams how you see fit and if you want to make another subteam called `lsp-codeowners` or something that seems like a good idea!

**Review by emu47 — APPROVED** [review]

File: standards/peer-review.md
Comment (EdwinGuzman) [inline standards/peer-review.md:18]: Something that we do in the DS repo is to add labels to PRs that need a review "Needs Review", need changes "Changes Requested", or are approved "Ship It". This helps us glance at the PR page and see what PRs we should review. I think we can suggest to add appropriate labels to PRs if they have been discussed within the team.

For example:
Screenshot 2024-04-17 at 4 21 42 PM

Comment (EdwinGuzman) [reply standards/peer-review.md:18]: This would be a responsibility of the requestor to add the initial label, and the reviewer to update the label when they complete their review.

**Review by EdwinGuzman — APPROVED** [review]
My comment is not a blocker.

**Comment by gioneill** [comment]
LGTM

**Review by nonword — APPROVED** [review]

File: standards/peer-review.md
Comment (mwbenowitz) [reply standards/peer-review.md:18]: I've added this as an option! I think this is a great idea for larger, more asynchronous teams (such as DS and eReading) but for some smaller teams may not be necessary. We should keep an eye on this and see if we want to make a firmer recommendation in the future as this specific issue was one of my motivations for updating this page.

**Pull request merged**
